nix/pkgs/libspdk/default.nix

```diff
@@ -122,7 +162,7 @@ in
       "--disable-unit-tests"
     ];
   });
-  debug = stdenv.mkDerivation (drvAttrs // {
+  debug = llvmPackages_11.stdenv.mkDerivation (drvAttrs // {
```

Comment:
These changes aren't strictly necessary, but they're steps on the way to cross compilation working.